bin/smoke

```diff
@@ -216,8 +224,7 @@ end
 
 begin
   @options, @args = parse_options
-  application = @args[0]
-  validate_application(application)
+  application = @args[0].split(/ /) # quick hack, does not consider blanks in file names
```

Comment:
There probably is, but adding a gem would make this harder to run. Right now you need zero, and I'd like to get it working as a compiled binary with no dependencies at all in the future.